reverseArray.js

```diff
@@ -0,0 +1,20 @@
+function transformArrayOfNumbers(array) {
+    //fliter and return odd numbers
+   const oddNumbers =  array.filter( (arrNumber)=> {
+        return arrNumber %2 == 1
```

Comment:
You should use the strict equality operator `===` here instead of `==`. 

Read more about [Strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality) here.